### PR TITLE
Quantum and css/jsschoolboardwiki groups

### PR DESCRIPTION
**Pull request by NDKilla**
https://meta.miraheze.org/wiki/Request_features#Quantum_.28Status:_Pending.29

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -876,6 +876,16 @@
 				'rollbacker',
 			),
 		),
+		'+cssandjsschoolboardwiki' => array(
+			'Founder' => array(
+				'autopatrolled',
+				'bot',
+				'bureaucrat'
```
Comment (Southparkfan) [inline LocalSettings.php]: Missing comma at end of file.

File: LocalSettings.php [hunk 2]
```diff
@@ -920,6 +930,16 @@
 				'admin',
 			),
 		),
+		'+quantumwiki' => array(
+			'Founder' => array(
+				'autopatrolled',
+				'bot',
+				'bureaucrat'
```
Comment (Southparkfan) [inline LocalSettings.php]: Missing comma at end of line.

File: LocalSettings.php [hunk 3]
```diff
@@ -1256,6 +1289,16 @@
 				'rollbacker',
 			),
 		),
+		'+cssandjsschoolboardwiki' => array(
+			'Founder' => array(
+				'autopatrolled',
+				'bot',
+				'bureaucrat'
```
Comment (Southparkfan) [inline LocalSettings.php]: Missing comma at end of line.

File: LocalSettings.php [hunk 4]
```diff
@@ -1299,6 +1342,16 @@
 				'admin',
 			),
 		),
+		'+quantumwiki' => array(
+			'Founder' => array(
+				'autopatrolled',
+				'bot',
+				'bureaucrat'
```
Comment (Southparkfan) [inline LocalSettings.php]: Missing comma at end of line.

**Comment by Reception123** [comment]
Should be fixed in #306

**Comment by NDKilla** [comment]
Duplicated and fixed in #306

**Pull request closed without merging**